csharp/src/Google.Protobuf/Reflection/ReflectionUtil.cs

```diff
@@ -71,7 +71,7 @@ static ReflectionUtil()
 
         /// <summary>
         /// Empty Type[] used when calling GetProperty to force property instead of indexer fetching.
-        /// </summary>
+        /// </summary>getFieldFunc
```

Comment:
typo?

Reply:
Looks like it. I have no idea how it could've gotten there.